guide/src/welcome.md

```diff
@@ -31,6 +31,8 @@ A suite of tutorials demonstrating how to do different things with Nannou. For
 example, "How do I output sounds?", "How do I draw shapes?", "How can I connect
 to my laser?"
 
+	- [Community Tutorials](/community_tutorials.md)
```

Comment:
This will end up formatted a little strangely - maybe instead we can add another sentence along the lines of

> You can find more tutorials by the community [here](./community_tutorials.md).